master/internal/config/authz_config.go

```diff
@@ -80,8 +80,9 @@ func (a AssignWorkspaceCreatorConfig) Validate() []error {
 
 // IsRBACUIEnabled returns if the feature flag RBAC should be enabled.
 func (c AuthZConfig) IsRBACUIEnabled() bool {
+	_, isRBACCapable := knownAuthZTypes["rbac"]
 	if c.RBACUIEnabled != nil {
-		return *c.RBACUIEnabled
+		return *c.RBACUIEnabled && isRBACCapable
```

Comment:
@corban-beaird was just making a change around this https://github.com/determined-ai/determined-ee/pull/944#discussion_r1260361111 

I'm not sure what rbac UI does. I need to lookup the docs